### ISSUE TITLE
Use server-sent events for long-running tasks

### ISSUE THREAD

**Issue by dermestid** [dermestid/bold-phylodiv-scripts#89]
instead of polling for updates or just waiting on a response from the initial request

obviously this only relates to web deployment of the full app, which as of #88 is limited to testing and hypothetical non-CLI users, so a low priority

**Comment by dermestid** [comment]
addressed in #90
